src/services/auth-services.ts

```diff
@@ -0,0 +1,37 @@
+// Libraries
+import axios from '@/axios-auth'
+
+/**
+ * Class that provides integration with the Auth API.
+ */
+export default class AuthServices {
```

Comment:
Changing these from mixins to classes:
- is cleaner since the methods have their own namespace instead of polluting `this`
- is easier to test
- is consistent with our newer UIs

Reply:
Looks much cleaner!

Reply:
As part of these future changes should we start trying to make the mixins as loosely coupled as possible with the Components that use them to prepare to change to the Compositon API?

Reply:
Yes, this change is heading in that direction, right?

Reply:
I think so, anything the creates greater distinction between the service/mixing and the component helps.